packages/ui/src/components/va-rating/components/VaRatingItem/VaRatingItem.vue

```diff
@@ -45,14 +48,15 @@ export default defineComponent({
     tabindex: { type: Number, default: 0 },
     disabled: { type: Boolean, default: false },
```

Comment:
Is it used anywhere?

Reply:
yes:
<img width="629" alt="image" src="https://user-images.githubusercontent.com/55198465/174727829-f33bc45f-0eb7-4a10-8e81-10ea4a5dab38.png">